VoiceControl/VoiceControl.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This looks to be the generated plugin config json file (generated from VoiceControl.config) which is incorrect. The VoiceControl.json needs to define the interface for this plugin. Pls refer to https://github.com/rdkcentral/rdkservices/blob/sprint/2201/DisplaySettings/DisplaySettings.json. Also the documentation (VoiceControl.md) needs to be generated from the interface file. Pls refer to https://github.com/rdkcentral/RDKServices#documentation